### DISCUSSION TITLE
LDAP authentication not working with Airflow 2.0?

### DISCUSSION THREAD

**Discussion by usernametakenforever** [apache/airflow#13840] [Q&A]
I have created webserver_config.py and added all the required configuration needed for LDAP authentication. I'm not able to login and logs show no error messages except for "login failed for user xxxxx". 

On the LDAP server, we never see any connection attempt being made by this new airflow server. Below is my configuration in webserver_config.py

```
import os
from airflow import configuration as conf
from flask_appbuilder.security.manager import AUTH_LDAP

basedir = os.path.abspath(os.path.dirname(__file__))

SQLALCHEMY_DATABASE_URI = conf.get('core', 'SQL_ALCHEMY_CONN')
CSRF_ENABLED = True
AUTH_TYPE = AUTH_LDAP
AUTH_ROLE_ADMIN = 'Admin'
AUTH_USER_REGISTRATION = True
AUTH_USER_REGISTRATION_ROLE = "Admin"
AUTH_LDAP_SERVER = 'ldaps://ldap_server:636'
AUTH_LDAP_SEARCH = "DC=xxxxx,DC=local"
AUTH_LDAP_BIND_USER = 'CN=xxx xxx,OU=!!NewYork,DC=xxxxxxx,DC=local'
AUTH_LDAP_BIND_PASSWORD = 'xxxxx'
AUTH_LDAP_UID_FIELD = 'sAMAccountName'
AUTH_LDAP_USE_TLS = False
AUTH_LDAP_ALLOW_SELF_SIGNED = False
AUTH_LDAP_TLS_CACERTFILE = '/etc/airflow/cert.pem'
```

Please let me know if any other information is needed

**Comment by ronaldzhanje** [comment]
When I work with LDAP I use tools like [LDAPSEARCH](https://devconnected.com/how-to-search-ldap-using-ldapsearch-examples/) to verify my LDAP query before I plug it into my code. Can you validate that the search query you are using returns the expected results? You also want to make sure that your certificate is valid.

**Reply by usernametakenforever** [reply]
Bump

**Reply by a8j8i8t8** [reply]
For me it's not working at all :-( I'm not sure if it's my LDAP or my settings. https://github.com/apache/airflow/discussions/14190

**Reply by usernametakenforever** [reply]
Yes same here. I have tried it all, LDAP doesn't work for me. I'm waiting if someone can confirm if it works on version 2.0 and above.
